lib/ES/TargetRepo.pm

```diff
@@ -0,0 +1,97 @@
+package ES::TargetRepo;
+
+use strict;
+use warnings;
+use v5.10;
+
+use Cwd;
+use Path::Class();
+use Encode qw(decode_utf8);
+use ES::Util qw(run sha_for);
+
+use base qw( ES::BaseRepo );
+
+my %Repos;
+
+#===================================
+# Create a repo for tracking *built* docs. This creation doesn't do anything
+# on disk and the repo should be prepared by calling `update_from_remote` and
+# `checkout_minimal`. The first call interacts with the network and can take
+# a long time, especially if the repo doesn't exist locally. The second call
+# is generally much quicker.
+#===================================
+sub new {
+#===================================
+    my ( $class, %args ) = @_;
+
+    $args{name} = 'target_repo';
+    my $self = shift->SUPER::new(%args);
```

Comment:
I think it is the same here `$class->SUPER`

Reply:
:+1: